### PR TITLE
ROE-16 Add makefile with sonar configuration

### PR DESCRIPTION
**Pull request by mattch1**
Added a make file to the application it has in it:
A build command that runs ci
and a clean command that removes the build and dist directory and the zip file as well as the log
and a lint and sonarqube command
and it also runs coverage
It then packages and builds the app

### CODE REVIEW

**Comment by mwestacott** [comment]
I don't think the .idea directory should be added to source control.

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,49 @@
+artifact_name       := overseas-entities
```
Comment (markpit) [inline Makefile:1]: i think artifact name should be overseas-entities-web

Comment (mouhajer-ch) [inline Makefile:1]: Probably should be `overseas-entities-web`?

Comment (mattch1) [reply Makefile:1]: See: https://github.com/companieshouse/confirmation-statement-web/pull/6/files

Comment (mattch1) [reply Makefile:1]: See: https://github.com/companieshouse/confirmation-statement-web/pull/6/files

Comment (markpit) [reply Makefile:1]: ah, looks like it was corrected since then - https://github.com/companieshouse/confirmation-statement-web/blob/main/Makefile

**Comment by mwestacott** [comment]
One more IDEA file to delete @mattch1 .

**Review by mwestacott — APPROVED** [review]

**Pull request merged**